lib/controller/page/create_idea_controller.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:idea_tracker/model/idea.dart';
+
+class CreateIdeaController extends ChangeNotifier {
```

Comment:
Rename to CreateIdeaPageController.

Largely unnecessary, but just in case we have a CreateIdeaDialog or CreateIdeaBottomSheet in the future, it can be confusing.